pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2024,6 +2024,38 @@ var _ = Describe("Converter", func() {
 			Entry("use virtio transitional", true),
 			Entry("use virtio non-transitional", false),
 		)
+		DescribeTable("Should set the error policy", func(epolicy v1.ErrorPolicy, expected string, specified bool) {
+			v1Disk := &v1.Disk{
+				Name: "mydisk",
+				DiskDevice: v1.DiskDevice{
+					Disk: &v1.DiskTarget{
+						Bus: v1.VirtIO,
+					},
+				},
+			}
+			if specified {
```

Comment:
nit - `if epolicy != nil` and drop `specified bool`?

Reply:
ErrorPolicy doesn't take nil as value, `if epolicy != nil` doesn't compile. I could make the ErrorPolicy a pointer or add an empty value constant but it seems a bit overkilled

Reply:
Apologies mistakenly assumed you were using pointers, we don't enforce their use for optional attributes in the API as much as I'd prefer us to so we could always use `nil` as an empty value.

Reply:
ok, good to know. I can update the PR accordingly, probably tomorrow :)

Reply:
@lyarwood I adjusted the code and now the `ErrorPolicy` in the VMI definition is a pointer